WcaOnRails/app/models/team_member.rb

```diff
@@ -51,6 +55,34 @@ def cannot_ban_user_with_upcoming_comps
     end
   end
 
+  def status
+    if leader?
+      TEAM_STATUS_LEADER
+    elsif senior_member?
+      TEAM_STATUS_SENIOR_MEMBER
+    else
+      TEAM_STATUS_MEMBER
+    end
+  end
+
+  def role
+    {
+      start_date: start_date,
+      is_active: current_member?,
+      group: {
+        id: UserGroup.group_types[:teams_committees] + "_" + team.id.to_s,
```

Comment:
No, what I meant is: Why do you even _need_ this prefix, when the same information is already present in `group_type`?

Reply:
If I don't have the prefix, I won't be able to understand that this is not actual `role`. It might get confused between team ID and group ID at some places.